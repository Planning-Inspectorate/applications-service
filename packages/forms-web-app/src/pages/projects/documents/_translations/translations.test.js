const enTranslations = require('./en.json');
const cyTranslations = require('./cy.json');

describe('pages/projects/documents/_translations', () => {
	it('should return the correct English projects documents translations', () => {
		expect(enTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh projects documents translations', () => {
		expect(cyTranslations).toMatchSnapshot();
	});

	it('should returning matching keys for both English and Welsh translations', () => {
		expect(Object.keys(enTranslations).sort()).toStrictEqual(Object.keys(cyTranslations).sort());
	});
});
