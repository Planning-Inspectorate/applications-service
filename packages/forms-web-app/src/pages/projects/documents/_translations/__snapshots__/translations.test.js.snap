// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/projects/documents/_translations should return the correct English projects documents translations 1`] = `
{
  "common": {
    "documents": "documents",
  },
  "errors": {
    "emptyValue": "The {{-formGroup}} date must include {{-emptyValue}}",
    "emptyValues": "{{-errorMessage}} and {{-emptyValue}}",
    "fromDateAfterToDate": "The from date entered should start before the to date",
    "fromDatePast": "The from date must be in the past",
    "mustBeRealDate": "The {{-formGroup}} date must be a real date",
    "title": {
      "from": "from",
      "to": "to",
    },
  },
  "filteredResults": {
    "heading1": "Filtered results",
  },
  "heading1": "Documents",
  "heading2": "Search documents",
  "noResults": {
    "linkText1": "Return to the project overview",
    "paragraph1": "There are no project application documents available to display at the moment.",
  },
  "noResultsFound": {
    "linkText1": "Clear search and filters",
    "paragraph1": "No results were found matching your search term or filters.",
    "paragraph2": "Would you like to clear your search and filters to view all available documents instead?",
  },
  "paragraph1": "{{-link}} containing document reference numbers",
  "paragraph1LinkText": "View examination library",
  "paragraph2": "Search by author, description or document type.",
  "paragraph3": "Showing {{-from}} to {{-to}} of {{-total}} documents, newest first.",
}
`;

exports[`pages/projects/documents/_translations should return the correct Welsh projects documents translations 1`] = `
{
  "common": {
    "documents": "dogfennau",
  },
  "errors": {
    "emptyValue": "Rhaid i'r {{-formGroup}} gynnwys {{-emptyValue}}",
    "emptyValues": "{{-errorMessage}} a {{-emptyValue}}",
    "fromDateAfterToDate": "Dylai'r dyddiad o a nodir ddechrau cyn y dyddiad i",
    "fromDatePast": "Rhaid i'r dyddiad o fod yn y gorffennol",
    "mustBeRealDate": "Rhaid i'r {{-formGroup}} fod yn ddyddiad go iawn",
    "title": {
      "from": "dyddiad o",
      "to": "dyddiad i",
    },
  },
  "filteredResults": {
    "heading1": "Canlyniadau wedi'u hidlo",
  },
  "heading1": "Dogfennau",
  "heading2": "Chwilio dogfennau",
  "noResults": {
    "linkText1": "Dychwelyd at Drosolwg y Prosiect",
    "paragraph1": "Nid oes dogfennau cais prosiect ar gael i'w harddangos ar hyn o bryd.",
  },
  "noResultsFound": {
    "linkText1": "Clirio chwiliad a dewisiadau hidlo",
    "paragraph1": "Ni ddaethpwyd o hyd i unrhyw ganlyniadau sy'n cyfateb i'ch term chwilio na'ch dewisiadau hidlo.",
    "paragraph2": "A hoffech chi glirio'ch chwiliad a'ch dewisiadau hidlo i weld yr holl ddogfennau sydd ar gael yn lle hynny?",
  },
  "paragraph1": "{{-link}} sy'n cynnwys rhifau cyfeirnod dogfennau",
  "paragraph1LinkText": "Edrychwch ar lyfrgell yr archwiliad",
  "paragraph2": "Chwilio yn Ã´l awdur, disgrifiad neu'r math o ddogfen.",
  "paragraph3": "Yn dangos {{-from}} i {{-to}} o {{-total}} o ddogfennau, gyda'r rhai mwyaf newydd yn gyntaf.",
}
`;
